lib/stack_master/commands/helper.rb

```diff
@@ -0,0 +1,21 @@
+require 'active_support/concern'
+
+module StackMaster
+  module Commands
+    module Helper
+      extend ActiveSupport::Concern
+
+      included do
+        def window_size
+          size = ENV["COLUMNS"] || `tput cols`.chomp
```

Comment:
Nitpick: I would prefer to use `ENV.fetch('COLUMNS') { .. }` for this behaviour.